src/openklant/components/klantinteracties/api/serializers/internetaken.py

```diff
@@ -68,20 +71,43 @@ class Meta:
             },
         }
 
+    def validate(self, attrs):
+        status = attrs.get("status", None)
+        if not status and self.instance:
+            status = self.instance.status
+
+        if attrs.get("afgehandeld_op") and status == Taakstatus.te_verwerken.value:
+            raise serializers.ValidationError(
+                {
+                    "afgehandeld_op": _(
+                        "De Internetaak kan geen afgehandeld op datum bevatten "
+                        "als de status nog in '{te_verwerken}' staat."
+                    ).format(te_verwerken=Taakstatus.te_verwerken.value)
```

Comment:
```suggestion
                        "als de status niet op '{verwerkt}' staat."
                    ).format(verwerkt=Taakstatus.verwerkt)
```

Reply:
@bart-maykin can you update the error message text as well so it's the same as the conditional?